src/main/java/com/binance/dex/api/client/Wallet.java

```diff
@@ -88,6 +97,9 @@ public synchronized void initAccount(BinanceDexApiRestClient client) {
     public synchronized void reloadAccountSequence(BinanceDexApiRestClient client) {
         AccountSequence accountSequence = client.getAccountSequence(this.address);
         this.sequence = accountSequence.getSequence();
+        if (this.usePreviousSequence && this.sequence < this.previousSequence) {
+            this.sequence = this.previousSequence;
+        }
     }
 
     public synchronized void increaseAccountSequence() {
```

Comment:
we may update previousSequence whenever sequence is changed(increased or decreased)

Reply:
you are right.